version/version.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 The Kube-burner Authors.
```

Comment:
Update header pls :)

Reply:
I kept it "as-is" so not to mess with "Kube-burner" Orginal Authors :P

Reply:
@krishvoor We have our own header for `go-commons` btw: https://github.com/cloud-bulldozer/go-commons/blob/main/comparison/elastic.go#L1 😛

Reply:
Updated, ptal